airbyte/_processors/sql/bigquery.py

```
@@ -203,7 +207,10 @@ def _ensure_schema_exists(
         if self._schema_exists:
             return
 
-        sql = f"CREATE SCHEMA IF NOT EXISTS {self.sql_config.schema_name}"
+        sql = (
+            f"CREATE SCHEMA IF NOT EXISTS `{self.sql_config.project_name}.{self.sql_config.schema_name}` "
```

Ruff (0.11.9)

Line too long (106 > 100)

(E501)